notebooks/Milestone4.ipynb

```diff
@@ -221,7 +221,15 @@
    "source": [
     "## 3. Summarize your journey from Milestone 1 to Milestone 4\n",
     "rubric={mechanics:10}\n",
-    ">There is no format or structure on how you write this. (also, no minimum number of words).  It's your choice on how well you describe it."
+    ">There is no format or structure on how you write this. (also, no minimum number of words).  It's your choice on how well you describe it.\n",
+    "\n",
+    "In Milestone 1, we learned the pain of dealing with big data on a local machine. We downloaded files using API and combined the data to create one extremely large rainfall dataset of over 62,000,000+ rows. To minimize memory usage while working with this large data set, we loaded the data in chunks of 10,000,000 and converted the datatypes to `float32`. Finally, we conducted a simple EDA using the optimized data set from CSV, which took about a minute to execute. To compare the CSVs versus more efficient data files formats, we timed the data loading and EDA using a parquet file and. For each machine, it took about 4 seconds. It was interesting to see how long each task ran on different machines. For example, while everyone on the team has a MacBook, it was clear that the 16-GB RAM M1 chip was superior in execution time.\n",
```

Comment:
To compare the CSVs versus more efficient data files formats, we timed the data loading and EDA using a parquet file<del> and </del>.